### DISCUSSION TITLE
管理用户找回密码

### DISCUSSION THREAD

**Discussion by Kbstsn** [lejianwen/rustdesk-api#49] [Q&A] [answered]
如果给admin用户的密码忘了，有办法能够重置吗？

**Comment by lejianwen** [comment]
现在没有这个功能，之后可以加个命令行重置密码

**Comment by lejianwen** [comment, marked as answer]
已添加此功能 [重置管理员密码](https://github.com/lejianwen/rustdesk-api?tab=readme-ov-file#%E9%87%8D%E7%BD%AE%E7%AE%A1%E7%90%86%E5%91%98%E5%AF%86%E7%A0%81)
